.github/workflows/mcore-tag-bump-bot.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   main:
     runs-on: ubuntu-latest
-    environment: main
+    environment: ko3n1g/ci/improve-vm-maintenance
```

Comment:
Please avoid using user name for Prod code

Reply:
ups, thanks!